web/README.md

```diff
@@ -56,4 +56,12 @@ If more than one target is specified, the last one will take precedence.
 
 Before running unit tests on KeymanWeb, first run `./build.sh` according to the instructions above.
 
-Once the build is complete, running `npm test` will run the unit testing suite on your local machine in-browser.  Alternatively, see `unit_tests/test.sh`, which the former command executes.
+Once the build is complete, running `npm test` will run the unit testing suite on your local machine in-browser.
+Alternatively, see `unit_tests/test.sh`, which the former command executes.
+
+### Debugging Unit Tests
+1. During development, to run a specific unit test, change the `it` to `it.only`.
+    You can also run all tests under a specific group with `describe.only`.
+2. From this directory, run `./unit_tests/test.sh -debug`
```

Comment:
Alternatively, from `web/` or any `web/` subdirectory,

```
npm run test -- -debug
```

The `--` part tells `npm` to funnel anything afterward to the script as the script's command-line parameters.

As long as it's run from somewhere within the `web/` folder's hierarchy, that line will always run from `web/`, as that's where `package.json` is.  As I'm usually in `web/source/`, that's what I usually end up using.

Reply:
Good to know. I've added this to the README